src/scenes/home/landing/hero/hero.css

```diff
@@ -0,0 +1,77 @@
+.heroContainer {
+  display: flex;
+  flex-flow: column;
+}
+
+.pageHeading {
+  background-color: #191f39;
+  color: #ffffff;
+  text-align: center;
+  padding: 15px;
+  width: 100%;
+  box-sizing: border-box;
+  margin: 0;
+}
+
+.btnContainer {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+}
+
+@media screen and (max-width: 462px) {
+  .btnContainer {
+    flex-flow: column;
+  }
+}
+
+.embedContainer {
+  width: 100%;
+  box-sizing: border-box;
+}
```

Comment:
The `.embedContainer` div should also have similar horizontal padding as recommended for the `.pageHeading` to create some space between it and the text to the left - `padding: 0 30px` should work.